.gitignore

```diff
@@ -7,6 +7,10 @@
 # Ignore bundler config
 /.bundle
 
+# Ignore RVM config files
+.ruby-gemset
+.ruby-version
```

Comment:
Keeping `.ruby-version` might be a good plan due to the issues you had with 1.9.3.  Can you add one if it doesn't exist?